package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "build": "gulp build",
+    "prepare": "npm run build",
```

Comment:
It looks like this will get called after `npm install` automatically. Should we therefore change `createReleaseTarball.sh` to not call it explicitly?

Reply:
Yes, that's true. The tarball contains the .js file even without the addition `npm run build`.